src/Core/Location/InputLocation.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import "./_input-location.scss";
+
+export default class InputLocation extends Component {
+    handleQueryUpdate = (event) => {
+        this.props.onChange(event.target.value)
```

Comment:
Using `currentTarget` would be better here. And also make a quick search about why we need to use `currentTarget` over `target`.